packages/vendure-plugin-multiserver-db-sessioncache/src/session-cache.ts

```diff
@@ -0,0 +1,16 @@
+import { VendureEntity, DeepPartial, CachedSession } from '@vendure/core';
+import { Column, Entity, Index } from 'typeorm';
+
+@Entity()
+export class MultiServerDbSessionCache extends VendureEntity {
+  constructor(input?: DeepPartial<MultiServerDbSessionCache>) {
+    super(input);
+  }
+
+  @Column('varchar', { nullable: false })
+  session!: string;
```

Comment:
I think we can use TypeOrm's JSON field, so we don't have to stringify and have some sort of type checking:
```suggestion
  @Column('simple-json', { nullable: false })
  session!: CachedSession;
```